README.md

```diff
@@ -1 +1,50 @@
-# DSCI_522_group_31
\ No newline at end of file
+# Online Purchasing Intention Predictor
+
+- author: Yazan Saleh
+- contributors: Mai Le, Tran Doan Khanh Vu, Jingjing Zhi
+
+A data analysis project for DSCI 522 (Data Science workflows) aiming to predict purchasing intentions of online shoppers.
+
+## Introduction
+
+In this project, the question we are interested in answering is: given visitor pageview and session information data for an e-commerce platform, is the visitor intending to make a purchase or not? This, in essence, is a binary classification problem where the target class, namely intent to purchase, can be `TRUE` or `FALSE`.
+
+With the rising popularity of online shopping, particularly in the wake of the 2020 coronavirus pandemic, there exists a strong growth opportunity for businesses employing e-commerce solutions. While increasing overall traffic to online stores is a critical first step, higher traffic does not always convert into increased sales. Online visitors may exit the site without making a purchase for a variety of reasons including: loading times, website layout, pricing, amongst others.
+
+Therefore, being able to predict purchasing intentions can be a valuable tool because it could potentially allow businesses to implement measures to better capture users with purchasing intention and thus try to convert them into actual revenue generators. Predicting purchasing intentions can also be useful for making more accurate forecasts about the business.
+
+The data set used in this project is the "Online Shoppers Purchasing Intention" dataset provided by the [Gözalan Group](http://www.gozalangroup.com.tr/) and used by Sakar, OC et al. in their 2018 analysis publsihed in [Neural Computing and Applications](https://link.springer.com/article/10.1007/s00521-018-3523-0). The data set was sourced from UCI's Machine Learning Repository (Dua and Graff 2020) at this [link](https://archive.ics.uci.edu/ml/datasets/Online+Shoppers+Purchasing+Intention+Dataset). The specific file used for the analysis found [here](https://archive.ics.uci.edu/ml/machine-learning-databases/00468/online_shoppers_intention.csv). Each row in the data set contains pageview and session information belonging to a different visitor that browsed [Columbia](https://www.columbia.com.tr), an online e-commerce platform based in Turkey. Each row also contains the target class `REVENUE`, a boolean flag indicating whether that user session contributed to revenue or not. Pageview data includes the type of pages that the visitor browsed to and the duration spent on each page. Session information includes visitor-identifying data such as browser information, operating system information as well visitor location and type.
```

Comment:
The **Introduction** part is quite long I think, should we break it starting from this paragraph into a sub-section with the heading "Our research" or something similar? And probably **Introduction** can be changed to **Motivation**?